docs/api/session.md

```diff
@@ -1091,6 +1091,55 @@ app.whenReady().then(() => {
 })
 ```
 
+#### `ses.setUSBProtectedClassesHandler(handler)`
+
+* `handler` Function\<string[]> | null
```

Comment:
The function takes an object instead of `string[]`.

Reply:
@zcbenz `string[]` here denotes the return value expected from the handler, not the value passed into the handler.

Reply:
See for example https://www.electronjs.org/docs/latest/api/session#sessetdevicepermissionhandlerhandler.